Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -131,6 +147,29 @@ grep -Fxq "$exportline2" ~/.profile 2>/dev/null || {
     fi
 }
 
+if $IS_WINDOWS_10_WSL; then
+	#add windowing support
+	WSL_profile1="export DISPLAY=:0.0"
+	WSL_profile2="export LIBGL_ALWAYS_INDIRECT=1"
+	if ! grep -Fx "$WSL_profile1" ~/.profile; then
+        echo $WSL_profile1 >> ~/.profile
+        eval $WSL_profile1
+	fi
+	if ! grep -Fx "$WSL_profile2" ~/.profile; then
+        echo $WSL_profile2 >> ~/.profile
+        eval $WSL_profile2
+	
+	#remove an incorrect eabi from path if WSL added it to path
+	WSL_wrong_eabi_path="export PATH=/opt/gcc-arm-none-eabi-4_9-2015q3/bin:\$PATH"
```

Comment:
Who adds this to the path?  Could you add that to the comment?

Reply:
It's in the wiki to remove it. I don't think it happens now but if anyone re-runs this script then it will fix it